Loyc-Slim.sln

```diff
@@ -0,0 +1,395 @@
+
```

Comment:
The new solution file has another advantage: it makes it easier for people who use Mono (me) to work with the Loyc source code. The projects that I excluded there don't build on my machine - or AppVeyor CI - either.